src/amo/components/Home/index.js

```diff
@@ -159,7 +176,18 @@ export class HomeBase extends React.Component {
               sort: SEARCH_SORT_POPULAR,
             },
           }}
-          loading={popularExtensions === null}
+          loading={resultsLoaded === false}
+        />
+
+        <LandingAddonsCard
+          addons={featuredCollection}
+          className="FeaturedCollection"
```

Comment:
We'll often prefix a classname of a child component with the parent component's name, so this should be `Home-FeaturedCollection`.